Mathlib/CategoryTheory/Sites/Equivalence.lean

```diff
@@ -301,6 +301,33 @@ instance : PreservesSheafification J F :=
   PreservesSheafification.transport (A := A) J
     ((equivSmallModel C).inverse.inducedTopology J) (equivSmallModel C).inverse B F
 
+variable [e.functor.IsCocontinuous J K] [e.inverse.IsCocontinuous K J]
+
+instance : e.functor.IsDenseSubsite J K where
```

Comment:
Regardless of my latest comment, these are probably still worth adding, maybe as lemmas instead of instances, to avoid loops. As my comment shows, it is often more natural to prove that the equivalence induces the expected topology, rather than showing that it's co/continuous. I suspect this is more often the case.